pkg/rook/static/rookBlockSpec.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: ceph.rook.io/v1
```

Comment:
is this used?

Reply:
ah nope
added for personal reference (this is what you need to change to/from), but it's not used in the code

Should I remove it?

Reply:
up to you. i didnt see you replacing the image for airgap which raised a red flag but then noticed it was unused.

Reply:
I'll add comments at the start of the file